test/Java10andUp/build.xml

```diff
@@ -0,0 +1,91 @@
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+-->
+
+<project name="Java10AndUp" default="build" basedir=".">
+	<taskdef resource='net/sf/antcontrib/antlib.xml'/>
+	<description>
+		Test Example
+	</description>
+	
+	<!-- set global properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/Java10andUp" />
+	<property name="BUILDINFO_FSROOT" value="${BUILD_ROOT}" />
+		
+	<!--Properties for this particular build-->
+	<property name="src" location="./src"/>
+	<property name="TestUtilities" location="../TestUtilities/src"/>
+	<property name="build" location="./bin"/>
+	<property name="transformerListener" location="../Utils/src"/>
+
+	<target name="init">
+		<mkdir dir="${DEST}" />
+		<mkdir dir="${build}"/>
+	</target>
+	
+	<target name="compile" depends="init" description="Using ${JAVA_VERSION} java compile the source " >
+		<echo>Ant version is ${ant.version}</echo>
+		<echo>============COMPILER SETTINGS============</echo>
+		<echo>===fork:				yes</echo>
+		<echo>===executable:			${compiler.javac}</echo>
+		<echo>===debug:				on</echo>
+		<echo>===destdir:				${DEST}</echo>
+
+				<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
+					<src path="${src}"/>
+					<src path="${TestUtilities}" />
+					<src path="${transformerListener}" />
+					<compilerarg line="--add-exports java.base/jdk.internal.reflect=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach.target=ALL-UNNAMED" />
+					<classpath>
+						<pathelement location="../TestConfig/lib/asm-all.jar" />
+						<pathelement location="../TestConfig/lib/testng.jar"/>
+						<pathelement location="../TestConfig/lib/jcommander.jar"/>
+					</classpath>
+				</javac>
+	</target>
+
+	<target name="dist" depends="compile" description="generate the distribution" >
+		<mkdir dir="${DEST}"/>
+		<jar jarfile="${DEST}/GeneralTest.jar" filesonly="true">
+			<fileset dir="${build}"/>
+			<fileset dir="${src}/../" includes="*.properties,*.xml"/>
+		</jar>
+		<copy todir="${DEST}">
+			<fileset dir="${src}/../" includes="*.xml" />
+			<fileset dir="${src}/../" includes="*.mk" />
+		</copy>
+	</target>
+	
+	<target name="build" >
+		<if>
+			<equals arg1="${JAVA_VERSION}" arg2="SE10"/>
```

Comment:
Could you please put SE100 here? I updated testKidGen code to check for SE100. Please wait until PR #778 is merged, so we can launch a personal build to verify existing build will not be broken. Thanks.

Reply:
Done.